.github/workflows/lint.yaml

```diff
@@ -13,22 +13,46 @@
 # limitations under the License.
 
 name: lint
-on: [pull_request]
+on:
+  pull_request:
+  pull_request_target:
+    types: [labeled]
 
 jobs:
   build:
-    name: lint
+    if: "${{ github.event.action != 'labeled' || github.event.label.name == 'tests: run' }}"
+    name: Run lint
     runs-on: ubuntu-latest
     steps:
+      - name: Remove PR Label
+        if: "${{ github.event.action == 'labeled' && github.event.label.name == 'tests: run' }}"
```

Comment:
`s/tests: run/lint: run/` 
on this line and again on line 35

Reply:
Let me talk with @jackwotherspoon about this. There's a special "tests: run" label we use elsewhere to prevent external changes from running without an owner looking at the change first.